panther_bringup/config/imu_config.yaml

```diff
@@ -1,18 +1,25 @@
 ImuFilterNodelet:
+  gain: 0.1
+  zeta: 0.0
+  mag_bias_x: 0.0
+  mag_bias_y: 0.0
+  mag_bias_z: 0.0
+  orientation_stddev: 1e-2
   publish_tf: false
   use_mag: false
-  use_magnetic_field_msg: false
+  use_magnetic_field_msg: true
   fixed_frame: imu_link
   stateless: false
+  remove_gravity_vector: false
 
 PhidgetsSpatialNodelet:
   data_interval_ms: 8 # supported data rates: 4 8 16 24 32 40 ... 1000 (in ms)
   publish_rate: 100.0 # optional param publish_rate, defaults to 0
   # serial: 123456  # optional param serial, default is -1
   # hub_port: 0 # optional param hub_port, used if connected to a VINT hub
   # frame_id: imu_link  # optional param frame_id, default is "imu_link"
-  # linear_acceleration_stdev: 0.002745862  # optional param linear_acceleration_stdev, default is 280ug
-  # angular_velocity_stdev: 0.000349  # optional param angular_velocity_stdev, default is 0.095 deg/s
+  linear_acceleration_stdev: 2.4e-1  # optional param linear_acceleration_stdev, default is 280ug
+  angular_velocity_stdev: 3.0e-2  # optional param angular_velocity_stdev, default is 0.095 deg/s
```

Comment:
These values seems to be quite large, where do they come from? I think that it would be best to use ones from datasheet. Especially `angular_velocity_stdev` seems to be large when compared to the value from wheels, from my experience usually IMU data is more accurate than wheels one (when orientation is considered). Did you try making a loop (marking a starting position and returning to it) with different EKF configurations (only wheel odom orientation, only IMU orientation, fused orientation ...) ?